### PR TITLE
Encoder and decoder String <---> PHP serialize

### PR DESCRIPTION
**Pull request by Shankhanil**
First version of string <--> PHP serialize. I am working on a more secure and better version. The upcoming versions will include integer <---> PHP serialize, and null <---> PHP serialize @nadchif

### CODE REVIEW

**Comment by Shankhanil** [comment]
@nadchif  have a look and merge the PR

**Comment by Shankhanil** [comment]
@nadchif any update on this one?

File: src/convertors/PHPSerialize.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    } else if (checkformat(input, 'i') ) {
+      const decode = input.split(':');
+      return decode[1];
+    } else {
+      console.error('invalid PHPSerial format');
+      return;
+    }
+  };
+    // Decoding PHP Serial to Float
+  const decodeFloatFromPHPSerial = (input) =>{
```
Comment (nadchif) [inline src/convertors/PHPSerialize.js:69]: Can we move the contents of this decode function to be part of `decodeFromPHPSerial` so we can have just one `PHP Serial to String/Int/Flt` Entry on the Context Menu

Comment (Shankhanil) [reply src/convertors/PHPSerialize.js]: I have updated the code as per review in the next commit.

**Comment by Shankhanil** [comment]
Have a look @nadchif

**Pull request merged**